sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -118,6 +118,8 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                             .AddAzureMonitorTraceExporter());
 
             builder.WithMetrics(b => b
+                            .AddAspNetCoreInstrumentation()
```

Comment:
Won't this add extra metrics in .NET 8/newer? Since this is a distro, we can add only the required ones, instead of adding them all, and then asking users to drop unwanted using views?

Reply:
Nice thought; will explore it. Need to research how to retrieve dropped metrics if the customer is interested in them.

Reply:
Users can just call AddMeter("missing-meter-name")